frontend/package.json

```diff
@@ -30,41 +28,41 @@
     "lodash": "^4.17.21",
     "micromatch": "^4.0.2",
     "morgan": "^1.10.0",
-    "next": "^10.0.5",
+    "next": "^11.0.0",
     "node-sass": "^4.13.0",
     "prop-types": "^15.7.2",
     "raw-loader": "^4.0.0",
-    "react": "^16.12.0",
-    "react-dom": "^16.12.0",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
     "react-double-scrollbar": "^0.0.15",
     "react-dropzone": "^11.2.3",
     "react-hot-toast": "^1.0.1",
     "react-media-hook": "^0.4.9",
     "react-social-login-buttons": "3.1.2",
     "semantic-ui-css": "^2.4.1",
-    "semantic-ui-react": "^0.88.2",
+    "semantic-ui-react": "^2.0.3",
     "slugify": "^1.4.6",
     "styled-jsx": "^3.4.4",
     "swr": "^0.3.11",
-    "url-loader": "^4.1.0"
+    "url-loader": "^4.1.0",
+    "webpack": "^5.0.0"
   },
   "devDependencies": {
     "@types/faker": "^5.1.0",
     "@types/stripe": "^7.13.25",
+    "@welldone-software/why-did-you-render": "^6.2.0",
     "cypress": "6.8.0",
     "cypress-fail-fast": "^2.0.0",
     "cypress-terminal-report": "^3.2.1",
     "eslint": ">=7.0.0",
-    "eslint-config-airbnb": "18.2.1",
+    "eslint-config-next": "^11.0.0",
     "eslint-config-prettier": "8.3.0",
     "eslint-import-resolver-alias": "^1.1.2",
     "eslint-plugin-import": "^2.23.4",
-    "eslint-plugin-jsx-a11y": "^6.4.1",
     "eslint-plugin-prettier": "3.4.0",
-    "eslint-plugin-react": "^7.21.5",
-    "eslint-plugin-react-hooks": "^1.7.0",
     "faker": "^5.1.0",
     "lint-staged": ">=10",
+    "mocha": "^3.5.0",
```

Comment:
Where is this coming from?

Reply:
cypress-terminal-report

Reply:
But nothing about that dependency changed and `mocha` is not a peer dependency of it. Why is it being added to our devDependencies?

Reply:
It's a peer dependency, you can remove it and the waring of missing peer dependency will appear when installing the packages.

Reply:
It doesn't seem to be a peer dependency in the most recent version at least. Even if we get a warning about peer dependencies it's not a good enough reason to add it to our devDependencies if we are not using mocha otherwise.

Reply:
Resolved via eadb08d.